_translator-files/README.md

```diff
@@ -1,17 +1,14 @@
 # Website docs translation:
 
-All the files for translators to work on are contained in a single directory: `_translator-files/po/YOUR-LANGUAGE/`. **Please do not edit any other files**. When adding a new language, all the necessary files will be automatically created and modified.
+The recommended way to translate the website documentation is by using [Weblate](https://hosted.weblate.org/projects/jamulus/). If you are comfortable using GitHub and .po file editors, and still want to contribute your translation via a Pull Request, the files for each language are contained in: `_translator-files/po/LANGUAGE/`. You can find a guide on .po file editors [here](https://jamulus.io/kb/2022/08/04/Translating-po-files.html). Please make your PR against the `next-release` branch.
 
-Proceed as follows:
+#####Some tips regarding translations:
```

Comment:
All the links/headlines need checking. They don’t render correctly on the GitHub UI. I think a space is missing here and for some links there‘s a space too much

Reply:
Should be fixed now.